web/templates/attendance/index.html.eex

```diff
@@ -0,0 +1,23 @@
+<h2>Listing Course for attendance</h2>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>Term</th>
+      <th>Course</th>
+      <th>Teachers</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <%= for offered_course <- @offered_courses do %>
+        <tr>
+          <td><%= offered_course.term.name %></td>
+          <td><%= offered_course.course.name %></td>
+          <td><%= Enum.reduce(offered_course.teachers, "", fn(teacher, out) ->
```

Comment:
what do you think of extracting this whole logic
```elixir
Enum.reduce(offered_course.teachers, "", fn(teacher, out) ->
  Enum.join([out, teacher.name, teacher.family_name], " ")
end)
```
into a function in the `AttendanceView` module?